docs/blog/2019-02-11-gatsby-themes-livestream-and-example/index.md

```diff
@@ -0,0 +1,59 @@
+---
+title: >
+  Gatsby Themes: Watch Us Build a Theme Live
+date: 2019-02-11
+author: Jason Lengstorf
+tags:
+  - themes
+  - livestream
+  - tutorials
+---
+
+One of the most exciting things we’re working on at Gatsby right now is themes.
+
+You can find more details in [the announcement post](/blog/2018-11-11-introducing-gatsby-themes/) and [why we think themes are important](/blog/2019-01-31-why-themes/), but the short version is this:
+
+**Gatsby themes allow you to focus only on the parts of the site and app building process that you _need to care about_ by abstracting the rest away into a package.**
+
+What this means in practical terms is that you can use Gatsby themes to create a site with:
+
+- `package.json` with 4 installed packages (`gatsby`, `react`, `react-dom`, and your theme)
+- `gatsby-config.js` with metadata and a reference to your theme
+- a folder full of Markdown files for content
+
+And _that’s it_.
+
+No code. No assembling plugins. Just writing content.
+
+## Learn how to build a Gatsby theme.
+
+Recently I had the pleasure of building a theme from scratch with Gatsby’s own [John Otander](https://twitter.com/4lpine) live on [my weekly livestream](https://twitch.tv/jlengstorf).
+
+<iframe id="ytplayer" type="text/html" width="720" height="405"
```

Comment:
Can you add a `title` attribute for this iframe to indicate where it goes (an accessibility best practice)? Something like "Theme livestream on YouTube" would work great.

Reply:
```suggestion
<iframe id="ytplayer" title="Theme livestream on YouTube" type="text/html" width="720" height="405"
```

Reply:
@marcysutton good call — thanks!